sunpy/util/datatype_factory_base.py

```diff
@@ -43,14 +44,15 @@ class BasicRegistrationFactory(object):
     def __init__(self, default_widget_type=None,
                  additional_validation_functions=[], registry=None):
 
-        if registry:
-            self.registry = registry
-        else:
+        if registry is None:
             self.registry = dict()
+        else:
+            self.registry = registry
 
         self.default_widget_type = default_widget_type
 
-        self.validation_functions = ['_factory_validation_function'] + additional_validation_functions
+        self.validation_functions = \
```

Comment:
rather than using line continuation I would prefer you put it in brackets, just as a style choice.